tests/unittests/ImageIO/ImageIOTest.mm

```diff
@@ -1233,12 +1233,12 @@ static CFURLRef getURLRefForOutFile(const wchar_t* filename) {
     int loopCount = 15;
     float delayTime = 0.5;
     NSDictionary* gifEncodeOptions = @{
```

Comment:
I did a somewhat too-broad replace for Starboard.h, seeing where I could keep it as only SmartTypes.h. I'll revert this file before CR close, but leaving this open for now since I don't want to wipe the history again.

---
In reply to: [72326111](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72326111) [](ancestors = 72326111)